0. Kirish/0.2 Tamoyillari.md

```diff
@@ -0,0 +1,32 @@
+# 0.2 Tamoyillari
+Ushbu qismda biz Me'yor tamoyillarini o'rganamiz. Ushbu tamoyillar me'yorga mos keladigan tizimning barcha komponentlariga taalluqlidir. Xo'sh ushbu komponentlar brokerlar, xizmatlar yoki taqdim etuvchilar bo'ladimi?
+
+## 0.2.0 Birinchi navbatda - Odamlar
+Ushbu tamoyilning asosiy g'oyasi odamlarni hisobga olgan holda me'yorga mos keladigan tizimlarni qurish va loyihalashdir. Nafaqat tizimdan foydalanmoqchi bo'lganlar, balki uni saqlab qoladigan va rivojlantiradigan odamlarni ham hisobga olgan holda me'yorga mos keladigan tizimlarni qurish va loyihalashdir.
+
+Tizim "Birinchi navbatda - Odamlar" tamoyiliga amal qilishi uchun u murakkablikdan ko'ra oddiylikni hurmat qilishi kerak. Oddiylik qayta yozish imkoniyatini beradi. Va, shuningdek, U tizimning umumiy namunasida haqiqiy murakkablilikni ta'minlash uchun aqliy fikrlash bilan qurilgan o'zgartirib bo'lmas tizimlarni loyihalash imkoyatini ham beradi.
+
+Shuningdek, Me'yor asosiy muhandislik tushunchasini tushunishga nisbatan ilg'or muhandislik tushunchalarini o'lchash tamoyillarini qo'llaydi. Sanoatdagi yangi muhandislar ertangi kunning yetakchilaridir. Agar ular biron bir tizimni kerakli vaqtda sotib olmasalar yoki tushunishmasalar, ular oxir-oqibat taslim bo'lishadi va uni qayta-qayta yozadilar.
```

Comment:
`The Standard also enforces the principles of measuring advanced engineering concepts against the understanding of mainstream engineers.`

Shu gapni o'zini tushunish qiyin ekan. Buni soddaroq qilib yozishlarini so'rab sizga aloqaga chiqaman.

Reply:
`The Standard simplifies leveraging advanced engineering concepts for mainstream engineers`

Mana shu gap soddaroq tuzilgan. Shuni tarjima qilib qoyish kerak.